Template.php

```diff
@@ -232,16 +232,16 @@ public function add_if_new($param, $value) {
           if (strpos($value, ',')) {
             $au = explode(',', $value);
             $this->add($param, formatSurname($au[0]));
-            return $this->add('first' . (substr($param, -1) == '1' ? '1' : ''), formatForename(trim($au[1])));
+            return $this->add(sanitize_string('first' . (substr($param, -1) == '1' ? '1' : ''), formatForename(trim($au[1]))));
```

Comment:
Should sanitize_string surround formatForename(...)?  otherwise there's only one parameter to this->add